<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlogEditor</class>
 <widget class="QMainWindow" name="BlogEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1218</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; text-decoration: underline;&quot;&gt;Blog/Tweet&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;-- Return</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>*Notice: Double Click to Select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>410</y>
      <width>155</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="commentDeletionButton_">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>410</y>
      <width>155</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete this Comment</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downloadButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>420</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Download File</string>
    </property>
   </widget>
   <widget class="QListWidget" name="groupListWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>71</y>
      <width>191</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="bloglistWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>361</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="tweetButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>420</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Tweet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="blogButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>420</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Blog</string>
    </property>
   </widget>
   <widget class="QPushButton" name="blogdeletionButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>420</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete this Blogs</string>
    </property>
   </widget>
   <widget class="QListWidget" name="commentlistWidget">
    <property name="geometry">
     <rect>
      <x>809</x>
      <y>72</y>
      <width>381</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="CommentEdit">
    <property name="geometry">
     <rect>
      <x>809</x>
      <y>270</y>
      <width>381</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QListWidget" name="mediaPosts">
    <property name="geometry">
     <rect>
      <x>585</x>
      <y>71</y>
      <width>211</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="userList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <zorder>groupListWidget</zorder>
   <zorder>bloglistWidget</zorder>
   <zorder>tweetButton</zorder>
   <zorder>blogButton</zorder>
   <zorder>blogdeletionButton</zorder>
   <zorder>commentlistWidget</zorder>
   <zorder>CommentEdit</zorder>
   <zorder>label</zorder>
   <zorder>returnButton</zorder>
   <zorder>label_2</zorder>
   <zorder>okButton</zorder>
   <zorder>commentDeletionButton_</zorder>
   <zorder>downloadButton</zorder>
   <zorder>mediaPosts</zorder>
   <zorder>userList</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1218</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
